src/hotspot/share/gc/shenandoah/shenandoahHeap.hpp

```diff
@@ -603,7 +603,7 @@ class ShenandoahHeap : public CollectedHeap {
   inline bool is_in_active_generation(oop obj) const;
   inline bool is_in_young(const void* p) const;
   inline bool is_in_old(const void* p) const;
-  inline bool is_old(oop pobj) const;
+  inline bool is_not_in_active_young_collection(oop obj) const;
```

Comment:
Needs a single line documentation comment, given its not very natural/intuitive semantics.

Reply:
Added a comment.